lib/anagrams.rb

```diff
@@ -0,0 +1,44 @@
+class Anagrams
+  def initialize(dictionary)
+    @dictionary = dictionary
+  end
+
+  def group_anagrams
+    anagram_group = {}
+    read_file.group_by do |word|
+      sorted_word = sort_and_validate(word)
+      anagram_group[sorted_word] ||= []
+      (anagram_group[sorted_word] << word)
+    end
+    anagram_group.values
+  end
+
+  def find_anagram(input_word)
+    result = []
+    word_characters = sort_and_validate(input_word)
+    read_file.each do |word|
+      result.push(word) if sort_and_validate(word) == word_characters
+    end
+    result
+  end
+
+  def to_s
+    group_anagrams.each do |anagram_block|
+      puts(anagram_block.join(' ')) if anagram_block.size > 1
```

Comment:
@zzokki81 `to_s` means return object string representation. When called I would expect to get a string not to do print those strings to the output.